test/ViewExtensionLibraryTests/ViewExtensionLibraryTests.csproj

```diff
@@ -140,23 +118,49 @@
-    <PropertyGroup>
-      <ErrorText>This project references NuGet package(s) that are missing on this computer. Use NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
-    </PropertyGroup>
-    <Error Condition="!Exists('..\..\src\packages\cef.redist.x64.3.3325.1758\build\cef.redist.x64.props')" Text="$([System.String]::Format('$(ErrorText)', '..\..\src\packages\cef.redist.x64.3.3325.1758\build\cef.redist.x64.props'))" />
-    <Error Condition="!Exists('..\..\src\packages\cef.redist.x86.3.3325.1758\build\cef.redist.x86.props')" Text="$([System.String]::Format('$(ErrorText)', '..\..\src\packages\cef.redist.x86.3.3325.1758\build\cef.redist.x86.props'))" />
-    <Error Condition="!Exists('..\..\src\packages\CefSharp.Common.65.0.1\build\CefSharp.Common.props')" Text="$([System.String]::Format('$(ErrorText)', '..\..\src\packages\CefSharp.Common.65.0.1\build\CefSharp.Common.props'))" />
-    <Error Condition="!Exists('..\..\src\packages\CefSharp.Common.65.0.1\build\CefSharp.Common.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\..\src\packages\CefSharp.Common.65.0.1\build\CefSharp.Common.targets'))" />
-    <Error Condition="!Exists('..\..\src\packages\CefSharp.Wpf.65.0.1\build\CefSharp.Wpf.props')" Text="$([System.String]::Format('$(ErrorText)', '..\..\src\packages\CefSharp.Wpf.65.0.1\build\CefSharp.Wpf.props'))" />
-    <Error Condition="!Exists('..\..\src\packages\CefSharp.Wpf.65.0.1\build\CefSharp.Wpf.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\..\src\packages\CefSharp.Wpf.65.0.1\build\CefSharp.Wpf.targets'))" />
-  </Target>
   <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
        Other similar extension points exist, see Microsoft.Common.targets.
   <Target Name="BeforeBuild">
-  </Target>
+  </Target>  
+  -->
   <Target Name="AfterBuild">
+    <ItemGroup>
+      <CefDeps Include="$(Pkgcef_redist_x64)\CEF\*" Condition=" '$(ProcessorArchitecture)' != 'x86' " />
+      <CefDeps Include="$(Pkgcef_redist_x86)\CEF\*" Condition=" '$(ProcessorArchitecture)' == 'x86' " />
+      <CefSharpWpf Include="$(PkgCefSharp_Wpf)\CefSharp\x64\*" Condition=" '$(ProcessorArchitecture)' != 'x86' " />
+      <CefSharpWpf Include="$(PkgCefSharp_Wpf)\CefSharp\x86\*" Condition=" '$(ProcessorArchitecture)' == 'x86' " />
+      <CefSharpCommon Include="$(PkgCefSharp_Common)\CefSharp\x64\*" Condition=" '$(ProcessorArchitecture)' != 'x86' " />
+      <CefSharpCommon Include="$(PkgCefSharp_Common)\CefSharp\x86\*" Condition=" '$(ProcessorArchitecture)' == 'x86' " />
+    </ItemGroup>   
+    <Copy SourceFiles="@(CefDeps)" DestinationFolder="$(OutputPath)\%(RecursiveDir)" />
+    <Copy SourceFiles="@(CefSharpWpf)" DestinationFolder="$(OutputPath)" />
+    <Copy SourceFiles="@(CefSharpCommon)" DestinationFolder="$(OutputPath)" />
```

Comment:
Was this copying not being done before?

Reply:
Because it is only needed for the test execution

Reply:
It shouldn't be needed for testing... AFAIK Cefsharp doesn't run at all under nunit and is disabled.

Reply:
@mjkkirschner On the viewextensionlibrarytests, I was getting errors on some of the tests looking for the CefSharp assembly and its dependencies, so I discovered that our current build process copy those assemblies to the output folder and that's how I got that those tests passed

Reply:
Just to clarify @aparajit-pratap , the copying is actually happening on our current process build, but after the migration to packagereference, this process just stopped, I've tried to look into another csproj if the copying was defined there and I couldn't find anything, so that's the reason I create those.
 
BTW @mjkkirschner , I said that it was required for the testing, because without those copy steps on csproj, the build was successful, but some tests of that project failed with the error of missing assembly for CefSharp